hibikase/PlaybackWidget.cpp

```diff
@@ -100,6 +114,28 @@ void PlaybackWidget::OnPlayButtonClicked()
         QMetaObject::invokeMethod(m_worker, "Stop");
 }
 
+void PlaybackWidget::OnTimeSliderMoved(int value)
+{
+    if (!m_time_slider->isSliderDown())
+        return;
+
+    // Update karaoke text but not audio when dragging slider
+    // (Jos doesn't like the scrubbing sound)
+    emit TimeUpdated(std::chrono::microseconds(value * 1000));
+}
+
+void PlaybackWidget::OnTimeSliderReleased()
+{
+    if (!m_worker)
+        return;
+
+    // Audio stopped while seeking, restart it
+    if (m_state != QAudio::State::ActiveState)
+        QMetaObject::invokeMethod(m_worker, "Play");
+
+    QMetaObject::invokeMethod(m_worker, "Seek", Q_ARG(std::chrono::microseconds, std::chrono::microseconds(m_time_slider->value() * 1000)));
```

Comment:
Consider creating a const variable for `std::chrono::microseconds(m_time_slider->value() * 1000)` on the line above so that this line won't be as long.